File: Frontend/src/app/DTOs/Test/TestQuestionResultDTO.ts
import { GeneralQuestionQuestionDTO } from "../GeneralQuestion/GeneralQuestionQuestionDTO";

export interface TestQuestionResultDTO {
  question: GeneralQuestionQuestionDTO;
  answer: string | null;
  wasRight: boolean;
  pointsScored: number;
  pointsPossible: number;
}

File: Backend/DTOs/Test/TestQuestionResultDTO.cs
namespace Learnz.DTOs;
public class TestQuestionResultDTO
{
    public GeneralQuestionQuestionDTO Question { get; set; }
    public string? Answer { get; set; }
    public bool WasRight { get; set; }
    public int PointsScored { get; set; }
    public int PointsPossible { get; set; }
}